snuba/consumer.py

```diff
@@ -16,6 +15,8 @@
 
 logger = logging.getLogger("snuba.consumer")
 
+# TODO: Remove this abstraction entirely and rely on StreamMessageParser subclasses
+# to deal with the Kafka specific fields that are needed when consuming a message.
```

Comment:
This takes a step towards that goal in that partition is not used anymore. Offset is still used and requires more thinking before removing that.